guides/common/modules/con_puppet-architecture.adoc

```diff
@@ -1,15 +1,30 @@
-[id="puppet_guide_puppet_architecture_{context}"]
+[id="Puppet_architecture_{context}"]
 = Puppet Architecture
 
-Puppet uses a server-client architecture with managed hosts running the Puppet agent and {Project} or any {SmartProxies} acting as a Puppet server.
-The Puppet agent enforces the declared state on managed hosts.
-It sends facts to the Puppet server, receives catalogs that are compiled from Puppet manifests, and sends reports to your {Project}.
-Differences between the declared state and the actual state are called _drifts_ in Puppet terminology.
+Puppet is built on client-server architecture.
```

Comment:
To me, this makes less sense than the original. The former tells me how Puppet operates while I cannot really understand why I am reading the suggested change.

Reply:
I was trying to elaborate on the internal workings of Puppet and how it's integrated with {Project} -- it seemed insufficient for a new user and a bit inaccurate before (based on feedback from engineers).
For example, it didn't seem clear to me from the original text if the catalog is the same as the define state of a host. Further, the ENC seemed like an important term and as such should be clarified how it's related to the overall Puppet flow (client-server communication).

Reply:
I was speaking for line 4 alone!

Reply:
Oh. Okay, reverting.